### ISSUE TITLE
Is it truly unsupervised or more of pretraining for quality estimation?

### ISSUE THREAD

**Issue by kartheekmedathati** [avinabsaha/ReIQA#7]
The paper seems to use linear regression to map the pre-trained embeddings to the visual quality. I am wondering if it is right to call it unsupervised learning for image quality estimation?

**Comment by avinabsaha** [comment]
Hi, Thanks for your interest in our work!  

We intend to refer to the representation learning framework as unsupervised. That is why our title says Unsupervised Learning **for** Image Quality Assessment. 

Also, you could refer to this work for this amazing [survey](https://arxiv.org/abs/2208.11296) on types of self/unsupervised methods. 

Thanks!

**Comment by kartheekmedathati** [comment]
Thanks for pointing the survey, not sure if that is relevant.  "Unsupervised representation learning" instead of "unsupervised learning" could have made it more clear.  When I read the title I was excited that some label free mechanism was figured out for quality estimation...

**Comment by avinabsaha** [comment]
@kartheekmedathati Our method can also be used label-free, but only in Full Reference Quality Estimation. We have not reported the results, but we have some internal results that show that in the FR/2AFC setting, just using the L1 distance between embeddings from the **Re-IQA-Quality-Aware** model can be useful for Quality estimation. Although the results may not be state-of-the-art across datasets, but are still usable. Also, using more image scales (>2) has improved performance in unsupervised settings!

**Comment by avinabsaha** [comment]
Closing this issue for now. Feel free to re-open to discuss further.
